### PR TITLE
Support Angular 9. Bump version (v0.4.0)

### LINKED ISSUE #7: Update the tested Angular application to the latest stable version

### PR DESCRIPTION
**Pull request by Farfurix**
Fixes https://github.com/DevExpress/testcafe-angular-selectors/issues/7

### Changes
1. Angular versions are separated in their own folders (the npm "postinstall" script is used). Angular 8 test case is added.
2. Angular documentation link is added to the `waitForAngular` error message.

### CODE REVIEW

File: src/wait-for-angular.js [hunk 1]
```diff
@@ -36,7 +60,8 @@ export default ClientFunction(ms => {
 
         pingTimeoutId = window.setTimeout(() => {
             clearTimeouts();
-            reject(new Error('Cannot find Angular in development mode.'));
+            reject(new Error(`Cannot find information about Angular components. Make sure that the tested application was deployed in 'development' mode.
```
Comment (Farfurix) [inline src/wait-for-angular.js]: @VasilyStrelyaev

File: src/angular-selector.js [hunk 2]
```diff
@@ -6,8 +6,23 @@ export default Selector(complexSelector => {
...
+    // NOTE: Angular version 9 or higher
+    const walkingNativeElementsMode = window.ng && typeof window.ng.getComponent === 'function';
+    // NOTE: Angular version 8 or lower
+    const walkingDebugElementsMode = window.ng && typeof window.ng.probe === 'function';
+
+    const isPageReadyForTesting = (walkingNativeElementsMode || walkingDebugElementsMode) &&
+        typeof window.getAllAngularRootElements === 'function';
+
+    if (!isPageReadyForTesting) {
+        throw new Error(`The tested page was built without Angular framework or it didn't load completely.
```
Comment (Farfurix) [inline src/angular-selector.js]: @VasilyStrelyaev

**Review by miherlosev — APPROVED** [review]

File: src/angular-selector.js (hunk 2 above)
Comment (VasilyStrelyaev) [reply src/angular-selector.js]: Checked by Dirk:

The tested page does not use Angular or did not load correctly. Use the 'waitForAngular' function to ensure the page is ready for testing.

File: src/wait-for-angular.js (hunk 1 above)
Comment (VasilyStrelyaev) [reply src/wait-for-angular.js]: Checked by Dirk:

Cannot find information about Angular components. The tested application should be deployed in development mode. For more information, see https://angular.io/guide/deployment.

**Review by VasilyStrelyaev — CHANGES_REQUESTED** [review]
👆

File: .gitignore [hunk 3]
```diff
@@ -47,8 +47,19 @@ Temporary Items
 .apdisk
 
 node_modules/*
+test/data/angularjs/node_modules/*
```
Comment (nlevman) [inline .gitignore:50]: Got failed on yarn/npm install because didnt fin this folders @Farfurix

Comment (nlevman) [reply .gitignore:50]: sh: line 0: cd: ./test/data/angularjs: No such file or directory
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! testcafe-angular-selectors@0.4.0 install-angularjs-deps: `cd ./test/data/angularjs && npm i`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the testcafe-angular-selectors@0.4.0 install-angularjs-deps script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

Comment (Farfurix) [reply .gitignore:50]: @nlevman

Thank you. I fixed it in `testcafe-angular-selectors@0.4.1`. Please update your version.

**Pull request merged**